gap/PolygonalComplexes/animating.gd

```diff
@@ -143,12 +143,15 @@ DeclareOperation( "SetVertexCoordinates3DNC", [IsSimplicialSurface, IsList, IsRe
 #! The 3D-coordinates of vertex <A>vertex</A> has to have the format <A>[x,y,z]</A>.
 #! If the format of the <A>coordinates</A> is not correct, then an error is shown.
 #! This can happen, if the NC version is used to store the 3D-coordinates.
-#! The NC-version does not check the coordinate format saved in the print record.
+#! The NC-version does not check the coordinate format saved in the print record. 
+#! The <A>GetAllVertexCoordinates3DNC</A> command returns all coordinates from the print record without checking them as a list.

```

Comment:
I think "method" would be a more appropriate word than "command".

Reply:
See [772bfc6](https://github.com/gap-packages/SimplicialSurfaces/pull/259/commits/772bfc640c06f07b2470aadb228b7fdb9ca35aa5)